healthcheck/go.mod

```diff
@@ -4,8 +4,11 @@ go 1.16
 
 require (
 	github.com/btcsuite/btclog v0.0.0-20170628155309-84c8d2346e9f
-	github.com/lightningnetwork/lnd v0.14.2-beta
 	github.com/lightningnetwork/lnd/ticker v1.1.0
+	github.com/lightningnetwork/lnd/tor v1.0.0
```

Comment:
Don't forget to bump healthcheck version too.

Reply:
Yes, thanks!